src/main/java/Deadline.java

```diff
@@ -0,0 +1,23 @@
+import java.time.LocalDate;
+
+/**
+ * Represents a task to be completed by a certain time
+ */
+public class Deadline extends Task {
+
+    /**
```

Comment:
Perhaps adding a short summary of the method? This can also apply to other methods that do not have a description in other places.

Reply:
thanks, not sure how I should describe the constructor or toString, do you have suggestions?